src/olympia/hero/migrations/0020_primaryhero_addon_primaryhero_promoted_group.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.20 on 2025-03-25 19:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+def copy_over_promotions(apps, schema_editor):
+    PromotedGroup = apps.get_model('promoted', 'PromotedGroup')
+    Hero = apps.get_model('hero', 'PrimaryHero')
+    for hero in Hero.objects.all():
+        hero.addon = hero.promoted_addon.addon
+        hero.promoted_group = PromotedGroup.objects.get(group_id=hero.promoted_addon.group.id)
```

Comment:
`hero.promoted_addon.group_id`

Reply:
This is accessing the old class `PromotedClass`,  where its structured as`group.id`